Runtime/BacktraceDatabase.cs

```diff
@@ -182,7 +192,7 @@ public void Reload()
 #if UNITY_SWITCH
```

Comment:
small nit: Maybe check the Switch condition before validating the configuration?